cmd/errtrace/main_test.go

```diff
@@ -415,6 +415,30 @@ func TestListFlag(t *testing.T) {
 	}
 }
 
+func TestOptoutLines(t *testing.T) {
+	fset := token.NewFileSet()
+	f, err := parser.ParseFile(fset, "", `package foo
+func _() {
+	_ = "line 3" //errtrace:skip
+	_ = "this line not counted" // errtrace:skip
+	_ = "line 5" //errtrace:skip // has a reason
+	_ = "line 6" //nolint:somelinter //errtrace:skip // stuff
+}`, parser.ParseComments)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var got []int
+	for line := range optoutLines(fset, f.Comments) {
+		got = append(got, line)
+	}
+	sort.Ints(got)
+
+	if want := []int{3, 5, 6}; !slicesEqual(want, got) {
```

Comment:
given this is used in tests that so we aren't perf sensitive, is `slicesEqual` worth adding over `reflect.DeepEqual`?